lib/beaker/host/unix.rb

```diff
@@ -64,7 +64,7 @@ def self.aio_defaults
       h = self.foss_defaults
       h['puppetserver-confdir'] = '/etc/puppetlabs/puppetserver/conf.d'
       h['puppetservice']  = 'puppetserver'
-      h['puppetbindir']   = '/opt/puppetlabs/agent/bin'
+      h['puppetbindir']   = '/opt/puppetlabs/agent/bin:/opt/puppetlabs/bin'
```

Comment:
Hmm just saw @jeffmccune's ticket. We need to talk about how to do this in a way that unblocks puppetserver, but doesn't break puppet.